rosbag2_cpp/include/rosbag2_cpp/writers/sequential_writer.hpp

```diff
@@ -77,6 +77,12 @@ class ROSBAG2_CPP_PUBLIC SequentialWriter
    *
    * \param storage_options Options to configure the storage
    * \param converter_options options to define in which format incoming messages are stored
+   * \throws runtime_error if 
+   * database directory already exists,
+   * failed to create database directory,
+   * no storage could be initialized,
+   * invalid bag splitting size given,
+   * max cache size less or equal 0 when snapshot mode is enabled.
```

Comment:
I am not sure if we want to list the all possible exception here, but at least indent is needed as below.

```suggestion
   * \throws runtime_error if database directory already exists,
   *   failed to create database directory, no storage could be initialized,
   *   invalid bag splitting size given,
   *   max cache size less or equal 0 when snapshot mode is enabled.
```

Reply:
I agree that introduces some maintenance hazard keeping doc and implementation in sync, but it could be practical to have it there.

I'd also be fine with just a hint at some general throw behavior here. 
So I'd leave it up to any reviewer as I am not too familiar with the style use throughout the rest of the codebase.